### PR TITLE
Use ldflags and ast rewrite to set version infos

### PR DESCRIPTION
**Pull request by donutloop**
Use ldflags and ast rewrite to inject version during build time.

### CODE REVIEW

**Comment by donutloop** [comment]
@seseloos Please review

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,27 @@
 .PHONY: all
 all: test install ## build APIKit and run tests
 
+PACKAGE_ROOT = github.com/ExperienceOne/apikit
+PACKAGE_VERSION = ${PACKAGE_ROOT}/internal/framework/version
+
+GIT_COMMIT=""
+GIT_BRANCH=""
+GIT_BRANCH_CLEAN=""
```
Comment (seseloos) [inline Makefile:9]: @donutloop `GIT_BRANC_CLEAN=""` isn't used within the Makefile and can be removed

**Review by seseloos — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (donutloop) [reply Makefile:9]: I gonna remove this unused variable from this makefile.

Comment (donutloop) [reply Makefile]: @seseloos done

**Pull request merged**